onnxruntime/test/server/unit_tests/server_configuration_test.cc

```diff
@@ -32,13 +36,15 @@ TEST(ConfigParsingTests, AllArgs) {
 TEST(ConfigParsingTests, Defaults) {
   char* test_argv[] = {
       const_cast<char*>("/path/to/binary"),
-      const_cast<char*>("--model"), const_cast<char*>("testdata/mul_1.onnx"),
+      const_cast<char*>("--model_path"), const_cast<char*>("testdata/mul_1.onnx"),
```

Comment:
Why const_cast is used at here?

Reply:
Im not quite sure, original test used **const_cast** so i followed along